### ISSUE TITLE
Unable to init new repo with error fork/exec /data/restic-0.16.4: permission denied

### ISSUE THREAD

**Issue by janreppien** [garethgeorge/backrest#181]
**Describe the bug**
When trying to initialize a new repository, with the following config:

![Screenshot from 2024-03-29 08-19-27](https://github.com/garethgeorge/backrest/assets/62404072/a884c619-a948-461e-8270-756fc35c6457)

I get this error:

![Screenshot from 2024-03-29 08-19-42](https://github.com/garethgeorge/backrest/assets/62404072/759571ff-ef38-47ed-8421-7a6cd93619ea)

`Operation failed: [unknown] failed to init repo: init failed: command "/data/restic-0.16.4 init --json" failed: fork/exec /data/restic-0.16.4: permission denied`

I use the following docker-compose.yaml:

```
version: "3.2"
services:
  backrest:
    image: garethgeorge/backrest
    container_name: backrest 
    volumes:
      - /mnt/media/docker_data/backrest/data:/data
      - /mnt/media/docker_data/backrest/config:/config
      - /mnt/media/docker_data/backrest/cache:/cache
      - /mnt:/mnt # mount your directories to backup somewhere in the filesystem
      - /mnt/backupA/rp4backup:/repos
    environment:
      - BACKREST_DATA=/data # path for backrest data. restic binary and the database are placed here.
      - BACKREST_CONFIG=/config/config.json # path for the backrest config file.
      - XDG_CACHE_HOME=/cache # path for the restic cache which greatly improves performance.
      - BACKREST_PORT=9898
    restart: unless-stopped
    ports:
      - 9898:9898
```

The folders ` /mnt/media/docker_data/backrest/data`, ` /mnt/media/docker_data/backrest/config`, ` /mnt/media/docker_data/backrest/cache` all have permission `777`.

**To Reproduce**
1. Start container with provided docker-compose.ymal.
2. Click on 'Add Repo'
3. Fill in Values from Screenshot above
4. Click ok 'Submit'
5. See error

**Expected behavior**
A new repository should be created

**Desktop (please complete the following information):**
 - OS: Ubuntu 23.10
 - Browser: Firefox 118.0.1 (64-bit)
 - Backrest Version: 0.15.1

**Additional context**
I run backrest on a Raspberry Pi 4 with Raspberry Pi OS (bullseye)

**Comment by garethgeorge** [comment]
Two possible issues jump to mind

 1) backrest may have issues with SELinux when downloading and installing restic (e.g. it can't execute files on a volume mount outside of the container). Make sure this is turned off or embed restic in your container image.
 2) Ensure that you chmod +x the restic binary (backrest should have done this on installation).

**Comment by janreppien** [comment]
Thanks for the quick reply!

@1. SELinux is not installed the RaspberryPi.

@2.  The data folder as well as all files are executable.

```
drwxrwxrwx 1 root root        0 Mar 27 12:35 .
drwxrwxrwx 1 root root        0 Mar 27 12:35 ..
-rwxrwxrwx 1 root root        0 Mar 27 12:35 install.lock
-rwxrwxrwx 1 root root       64 Mar 27 12:35 jwt-secret
-rwxrwxrwx 1 root root    32768 Mar 30 08:17 oplog.boltdb
-rwxrwxrwx 1 root root 23396352 Mar 27 12:35 restic-0.16.4
```

I think I will just keep using the restic cli to back up my data and test backrest again, with the next release. Regardless thank you and all other contributors for this amazing project, I am looking forward to using it in the future :)

**Comment by garethgeorge** [comment]
Just released 0.16.0 which bundles backrest in the container image, you'll want to use `garethgeorge/backrest:scratch` most likely on a raspberry PI. Go ahead and give that a go and please reopen this bug if you're still having issues.
